cogito/gchatsink.go

```diff
@@ -0,0 +1,110 @@
+package cogito
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/Pix4D/cogito/googlechat"
+	"github.com/hashicorp/go-hclog"
+)
+
+// TODO FIND A BETTER PLACE?
```

Comment:
No strong opinion on that one, but given it's private and used in a single place, I think I would have just inlined it in `shouldSendToChat`.

Should it become a tunable, it can be moved to were the parsing takes place as it would become the default.

Reply:
Yes, exactly for both comments. In this PR I will just inline, and in the next ticket I will move close to the parsing.

Reply:
Done.